ThunderCloud/NavigationController.swift

```diff
@@ -73,6 +73,10 @@ public extension UINavigationController {
     /// Returns a shared instance of `UINavigationController`
     static let shared: UINavigationController = UINavigationController()
     
+    /// Defines the video formats that are supported within Storm apps.
+    /// While the MIME type of video/mp4 is checked when uploading, m4v videos can also have this MIME type.
+    private static let supportedVideoFormats: [String] = ["mp4", "m4v"]
```

Comment:
Unsure whether this makes more sense on `UINavigationController` or `Video`?

Reply:
Yeah this is true. Let me move it to Video! :D

Reply:
@simonmitchell updated in 26cea8bbe93fe6c74dd8954e5d9aceaecbf92f47 :D